src/main/java/tissue/window/MainWindow.java

```diff
@@ -0,0 +1,48 @@
+package tissue.window;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.VBox;
+import tissue.Tissue;
+
+/** Controller for the main GUI. */
+public class MainWindow {
+    @FXML private ScrollPane scrollPane;
+    @FXML private VBox dialogContainer;
+    @FXML private TextField userInput;
+    @FXML private Button sendButton;
+
+    private Tissue tissue;
+
+    private final Image userImage = new Image(this.getClass().getResourceAsStream("/images/user.png"));
+    private final Image tissueImage =
```

Comment:
Good variable naming! However these variables are perhaps constants and require ALLCAPS?